KsApi/models/Project.Country.swift

```diff
@@ -1,3 +1,6 @@
+private let eurMaxPledge: Int = 8_500
```

Comment:
Moved these to local vars seeing as EUR is used for several countries in the list.